### PR TITLE
fix: Indicate segments where points are disabled in Timetrack Panel (regression)

### LINKED ISSUE #2716: Indicate segments where points are disabled in Timetrack Panel
In old TimeTrack (synfig 1.4.x and earlier) the time spans where a Dynamic List entry is kept off is indicated by hatches.
See it on the left part of screenshot -
screenshot_001

We need to indicate those regions in new Timetrack Panel.

I suggest to indicate those regions with darker color (black color with 50% alpha) -
screenshot_002

### PR DESCRIPTION
**Pull request by rodolforg**
Fix #2716

### CODE REVIEW

File: synfig-studio/src/gui/widgets/widget_timetrack.cpp [hunk 1]
```diff
@@ -862,9 +862,39 @@ void Widget_Timetrack::draw_active_point_status(const Cairo::RefPtr<Cairo::Conte
...
+		} else {
+			if (initial_off_time != synfig::Time::end()) {
+				cr->rectangle(+w/2+time_plot_data->get_pixel_t_coord(initial_off_time), row_info.get_geometry().y, -w+time_plot_data->get_delta_pixel_from_delta_t_coord(activepoint.get_time() - initial_off_time), row_info.get_geometry().h);
+				cr->fill();
+				initial_off_time = synfig::Time::end();
+			}
+		}
+	}
+	if (initial_off_time != synfig::Time::end()) {
+		int initial_pixel_index = +w/2+time_plot_data->get_pixel_t_coord(initial_off_time);
```
Comment (ice0) [inline synfig-studio/src/gui/widgets/widget_timetrack.cpp:891]: `+w`? What does it mean?

Comment (rodolforg) [reply synfig-studio/src/gui/widgets/widget_timetrack.cpp:891]: The green or red line width

**Comment by rodolforg** [comment]
I just found out that OFF point only means OFF in that time point o.O Right after it is ON again...

**Comment by rodolforg** [comment]
> I just found out that OFF point only means OFF in that time point o.O Right after it is ON again...

Now it's properly working :)

**Review by ice0 — APPROVED** [review]

**Comment by ice0** [comment]
Merged. Thank you!

**Pull request merged**